base/linalg/dense.jl

```diff
@@ -587,6 +619,23 @@ inverting dense ill-conditioned matrices in a least-squares sense,
 
 For more information, see [^issue8859], [^B96], [^S84], [^KY88].
```

Comment:
@kshyatt, should the `For more information...` group with the other references after the example, should the other references after the example group with the `For more information...` before the example, or neither? Thoughts? Thanks!